.env

```diff
@@ -0,0 +1,5 @@
+PORT=9000
+DB_ENV=development
+JWT_SECRET='Vogon poetry is described as the third worst poetry in the Universe behind that of the Azgoths of Kria - four members of an audience died'
+DATABASE_URL='postgres://iodsnxveuforop:cec894cad044f67707fec504700dbfbdb908d3ebb8788a6ea5371f47015f6698@ec2-54-197-234-117.compute-1.amazonaws.com:5432/df29u1v792haqc'
+CLOUDINARY_URL=cloudinary://378729149483132:ngCHP3i8h2zG20iOKPI_rN5O54k@dyfighlzd
```

Comment:
This stuff should _not_ be up on GitHub, your `.env` should be in your `,gitignore`.